opm/autodiff/SimulatorFullyImplicitBlackoil_impl.hpp

```diff
@@ -102,10 +102,12 @@ namespace Opm
         const parameter::ParameterGroup param_;
 
         // Parameters for output.
-        bool output_;
-        bool output_vtk_;
-        std::string output_dir_;
-        int output_interval_;
+        const bool output_;
```

Comment:
I adjusted the code such that the parameters are only accessed when output is true.
I still think that avoiding uninitialized values is a good idea.